### PR TITLE
Make it easier to audit all sites in a network

### PR DESCRIPTION
**Pull request by mboynes**

### CODE REVIEW

File: inc/class-cli.php [hunk 1]
```diff
@@ -128,16 +115,36 @@ function( $post ) use ( &$errors, &$confirmed, &$queue ) {
...
-			$queue = [];
-
 			$confirmed += $batch_confirmed;
 
 			foreach ( $batch_errors as $post_id => $reason ) {
 				$errors[] = compact( 'post_id', 'reason' );
 			}
 		}
 
+		return compact( 'confirmed', 'errors' );
```
Comment (joshkadis) [inline inc/class-cli.php:125]: i gotta remember to use `compact()` more often

File: inc/class-cli.php [hunk 2]
```diff
@@ -128,16 +115,36 @@ function( $post ) use ( &$errors, &$confirmed, &$queue ) {
...
+	 *
+	 * [--fix]
+	 * : If present, script will attempt to fix any posts that fail the audit.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     wp jetpack sync-ep audit_posts
+	 */
+	public function audit_posts( $args, $assoc_args ) {
+		$fix = ! empty( $assoc_args['fix'] );
```
Comment (joshkadis) [inline inc/class-cli.php:141]: Something I picked up in my travels that I really like is prefixing booleans with `should` or `is`, i.e. `$should_fix`

**Review by joshkadis — APPROVED** [review]
➿

File: inc/class-cli.php (hunk 2 above)
Comment (mboynes) [reply inc/class-cli.php:141]: I like that! thanks for sharing

**Pull request merged**